layouts/Footer.tsx

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import { textFont, titleFont } from "@/pages/_app";
+import FooterNaviLink from "@/components/FooterNaviLink";
+import styles from "@/styles/Footer.module.css";
+import Image from "next/image";
+import fylla from "@/public/images/fyllaIcon.png";
+import badgeLN from "@/public/images/badge-ln.png";
+import badgeIN from "@/public/images/badge-in.png";
+import badgeFB from "@/public/images/badge-fb.png";
```

Comment:
No need to import image resources, you can import them inline, it can be more short and clear.

such as:

`<img src="img/featured-work-sec-img-1.png" alt="Dangcing Stars" />
`

Reply:
But in the nextjs document, they suggest using <Image /> rather than <img/>. I first use <img/>, and my IDE also suggests using <Image />.
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/110534329/219324941-0e6f57ae-33ab-4fae-bbfc-dc04a4a3777d.png">

Reply:
It seems that you IDE suggest you to use Image component from Next framework, but in pervious meetings, we have cleared that we should use img. In addition, I do not know the nextjs document suggestion, can you clarify it?

Reply:
Yes, I did not attend last week's meeting. I have changed them to <img />.